rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -597,11 +597,23 @@ def _maybe_enrich_transfers(  # pylint: disable=no-self-use
         if tx_log.topics[0] == GNOSIS_CHAIN_BRIDGE_RECEIVE and tx_log.address == '0x88ad09518695c6c3712AC10a214bE5109a655671':  # noqa: E501
             for event in decoded_events:
                 if event.event_type == HistoryEventType.RECEIVE:
+                    try:
+                        crypto_asset = event.asset.resolve_to_crypto_asset()
+                    except (UnknownAsset, WrongAssetType) as e:
+                        log.error(
+                            f'Failed to resolve asset {event.asset} in gnosis bridge event decoding. {str(e)}',  # noqa: E501
+                        )
+                        self.msg_aggregator.add_error(
+                            f'Could not identify bridged asset {event.asset}. Make sure that it '
+                            f'has all the required properties (name, symbol and decimals) and try'
+                            f'to decode again the event {event.event_identifier.hex()}.',
+                        )
```

Comment:
This makes no sense. You are asking the user for a call to action here but I do not understand it. This will happen automatically, even on the background while the user is doing something else.

And then he sees this message.  When can that happen? I would just add more details to `log.error()` and skip the event. If they end up missing stuff then we can tell them to look in the logs and they can see why.

```suggestion
                        self.msg_aggregator.add_error(
                            f'Could not identify bridged asset {event.asset}. Make sure that it '
                            f'has all the required properties (name, symbol and decimals) and try '
                            f'to decode the event again {event.event_identifier.hex()}.',
                        )
```

Perhaps in the future we  could have a specific type of message that is for decoding calls to action? Made an issue for this: https://github.com/rotki/rotki/issues/4965